core/backend/src/test/integration/HubUtility.ts

```diff
@@ -348,7 +348,7 @@ export class HubUtility {
 
   /** Push an iModel to the Hub */
   public static async pushIModel(requestContext: AuthorizedClientRequestContext, projectId: string, pathname: string, iModelName?: string, overwrite?: boolean): Promise<GuidString> {
-    assert.isTrue(HubMock.isValid, "Must use HubMock for tests that create iModels");
+    assert.isTrue(HubUtility.allowHubBriefcases || HubMock.isValid, "Must use HubMock for tests that create iModels");

```

Comment:
Why? We don't want to create any tests that don't use mocks - that's why I added this check. Please remove this.

Reply:
I kept this to just quickly test it with the Hub if we needed to - it seemed quite useful - e.g., used it to push some iModels to the Hub the other day. 

Also, we do have this flag elsewhere?

Reply:
i don't want to encourage people to write tests that write to iMH.

Reply:
i removed this stuff